### DISCUSSION TITLE
Using Multiple Devices results in error 403

### DISCUSSION THREAD

**Discussion by Mardon85** [ClassicOldSong/moonlight-android#21] [Q&A] [answered]
I've been playing with your software tonight and have been really impressed with it. I've got a number of android devices and a laptop.
I love the auto resolution switching feature. However it only works with one device, any subsequent devices fail to connect.

If I deregister all devices the the first one I re-register works fine but the others fail with: Connection Error "Failed to start Virtual Display (error 403). This behaviour is the same irrespective of whether it is the laptop or an android device.

**Comment by ClassicOldSong** [comment, marked as answer]
https://github.com/ClassicOldSong/Apollo/wiki/Permission-System

**Comment by Mardon85** [comment]
That did it, thanks for the awesome software and fast response!
